api/user_handler.go

```diff
@@ -0,0 +1,38 @@
+package api
+
+import (
+	"github.com/alphagov/paas-accounts/database"
+	"github.com/labstack/echo"
+	"net/http"
+)
+
+func UserHandler(db *database.DB) echo.HandlerFunc {
+		return func(c echo.Context) error {
+			var userUpdate database.User
+			err := c.Bind(&userUpdate)
+			if err != nil {
+				return err
+			}
+
+			userUpdate.UUID = c.Param("uuid")
+
+			req := c.Request()
+
+			if req.Method == "PATCH" {
```

Comment:
We should check the user exists before trying to PUT or PATCH, and return 404 if it doesn't.